webrender/src/platform/windows/font.rs

```diff
@@ -355,7 +355,10 @@ impl FontContext {
         } else {
             None
         };
-        let (_, _, bounds) = self.create_glyph_analysis(font, key, size, transform, bitmaps);
+        let (_, _, bounds) = match self.create_glyph_analysis(font, key, size, transform, bitmaps) {
```

Comment:
nit: You could do `.ok()?;` on the `Result` instead of the match statement here

Reply:
Made a substantial rewrite of GlyphRasterResult into a Result type to make this all much cleaner.